index.html

```diff
@@ -113,6 +113,29 @@ <h4 class="panel-title">
                       </div>
                     </div>
                   </div>
+
+                  <!-- Start add -->
```

Comment:
You don't need these comments, they make no sense once merged

Reply:
Corrected.